docs/source/examples/de_sscan.ipynb

```diff
@@ -0,0 +1,1542 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# The synApps `sscan`\n",
+    "\n",
+    "The synApps `sscan` record is used to measure scans of detector(s) *v*. positioner(s).  \n",
+    "The `sscan` record has three different scan modes:\n",
+    "\n",
+    "`sscan` scan mode | bluesky plan | description\n",
+    "--- | --- | ---\n",
+    "linear | `scan()` | step scan with constant-size steps\n",
+    "table | `list_scan()` | step scan with list of positions\n",
+    "fly | `fly()` | continuous motion scan\n",
+    "\n",
+    "**Goals**: Demonstrate use of the `sscan` record with Bluesky.\n",
+    "\n",
+    "1. Press SCAN button of a preconfigured scan.\n",
+    "\n",
+    "    1. Use a polling loop to wait for scan to end.\n",
+    "    2. Use a Status object to wait for scan to end.\n",
+    "\n",
+    "2. Same, but connect with most `sscan` record fields and get data after scan.\n",
+    "3. Setup the same scan from bluesky.\n",
+    "4. Add the scan data as a bluesky run.\n",
+    "\n",
+    "This notebook is intended for those who are familiar with EPICS and its motor, scaler, and sscan records but are new to Bluesky."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Python setup\n",
+    "\n",
+    "The first example will need only the bluesky RunEngine.  It will not need any\n",
+    "databroker catalog.  The EPICS IOC has a prefix of `gp:`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import bluesky\n",
+    "import bluesky.plan_stubs as bps\n",
+    "\n",
+    "RE = bluesky.RunEngine()\n",
+    "IOC = \"gp:\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Press SCAN button of a preconfigured scan\n",
+    "\n",
+    "Consider the case where the `sscan` record is already configured with all the\n",
+    "parameters necessary to perform a complete step scan.\n",
+    "\n",
+    "In the next figure, (a) shows `gp:scan1` is configured to step scan motor `m1`\n",
```

Comment:
It is esthetic, but I find the construction "(a) shows X is configured to... " a bit weird, it took me 2 read to understand (first read I thought there was a typo).
I would rather say:
"(a) shows the record `gp:scan1` configured to...  (b) shows `gp:scaler1`  configured with..."
or maybe:
"In (a), the record `gp:scan1` is configured to...  In (b), `gp:scaler1` is configured with..."

Reply:
Removed extra `is` words.  Started each sentence like `Figure (1a) shows ...`